simple/template/src/app.rs

```diff
@@ -32,8 +34,9 @@ impl App {
     /// Renders the user interface.
     ///
     /// This is where you add new widgets. See the following resources for more information:
+    ///
     /// - <https://docs.rs/ratatui/latest/ratatui/widgets/index.html>
-    /// - <https://github.com/ratatui/ratatui/tree/master/examples>
+    /// - <https://github.com/ratatui/ratatui/tree/main/ratatui-widgets/examples>
     fn draw(&mut self, frame: &mut Frame) {
```

Comment:
I've been renaming this render in a few places recently, as it's run from terminal::draw as the "render callback", so it makes sense to use a consistent name. WDYT?

Reply:
Hmm, do you suggest that we rename this to `render`? Not sure if I get the suggestion

Reply:
yeah - I think it makes sense to name methods which are passed to the terminal::draw method `render`, even though that clashes with the `Widget::render()` method.

Reply:
done!